postcss.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    plugins: [
+        require('tailwindcss'),
+        require('@fullhuman/postcss-purgecss')({
```

Comment:
Should we skip this on dev builds? Could be a simple `process.env.NODE_ENV !== production`

Unless rollup will recompile the CSS on any HTML change?